tests/ui/extenv/extenv-not-env.rs

```diff
@@ -0,0 +1,6 @@
+// run-pass
```

Comment:
`PATH` may not always be defined on all platform (like wasm, if we run tests on it), we should probably use a manually set variable.
```suggestion
// run-pass
// rustc-env:MY_ENV=/
```

Reply:
Oh I didn't know that argument. I was hesitating to run it only on linux but your approach is better.